idseq/uploader.py

```diff
@@ -35,19 +37,31 @@ def source_type(self):
     def parts(self, max_part_size):
         # Check if any file is over max_part_size and, if so, chunk
         if self.source_type() == 'local' and os.path.getsize(
-                self.path) > max_part_size * 1048576:
+                self.path) > max_part_size:
             part_prefix = self.path + PART_SUFFIX
-            print("splitting large file into {} MB chunks...".format(
-                max_part_size))
-            subprocess.check_output(
-                "split -b {}m {} {}".format(max_part_size, self.path,
-                                            part_prefix),
-                shell=True)
-            return subprocess.check_output(
-                "ls {}*".format(part_prefix), shell=True).decode("utf-8").splitlines()
+            return self.split_file(max_part_size, part_prefix)
         else:
             return [self.path]
 
+    def split_file(self, max_part_size, prefix):
+        print("Splitting large file into {} MB chunks...".format(int(max_part_size // 1E6)))
```

Comment:
since you mentioned the base 10 vs base 2, maybe you could comment that distinction here

Reply:
Done